src/view/utils/fetchFromPlatform.js

```diff
@@ -22,11 +23,14 @@ const IMS_HOST_PREFIX_STAGING = "ims-na1-stg1";
 const ERROR_CODE_INVALID_ACCESS_TOKEN = "401013";
 const ERROR_CODE_USER_REGION_MISSING = "403027";
 
-const ERROR_UNEXPECTED_SERVER_RESPONSE = "Unexpected server response.";
+const ERROR_UNABLE_TO_CONNECT_TO_SERVER =
+  "A connection to the server could not be established.";
+const ERROR_UNEXPECTED_SERVER_RESPONSE =
+  "An unexpected server response was received.";
 const ERROR_INVALID_ACCESS_TOKEN = "Your access token appears to be invalid.";
 const ERROR_NO_AEP_ACCESS =
-  "Your user account is not enabled for AEP access. Please contact your organization administrator.";
-const ERROR_RESOURCE_NOT_FOUND = "Resource not found.";
+  "Your organization is not provisioned for Adobe Experience Platform. Please contact your organization administrator.";
```

Comment:
As far as I can tell from Slack conversations, this scenario ("user region is missing") is because the org isn't properly provisioned and not so much of a user account thing.